SUMMARY.md

```diff
@@ -3,28 +3,32 @@
 ## [README](/README.md)
 
 ## [Basics](docs/basics/SUMMARY.md)
-  * [Epics](docs/basics/Epics.md)
-  * [Setting Up The Middleware](docs/basics/SettingUpTheMiddleware.md)
+
+* [Epics](docs/basics/Epics.md)
+* [Setting Up The Middleware](docs/basics/SettingUpTheMiddleware.md)
 
 * [Sponsors and Backers](/BACKERS.md)
 
 ## [Recipes](docs/recipes/SUMMARY.md)
-  * [Cancellation](docs/recipes/Cancellation.md)
-  * [Error Handling](docs/recipes/ErrorHandling.md)
-  * [Injecting Dependencies Into Epics](docs/recipes/InjectingDependenciesIntoEpics.md)
-  * [Writing Tests](docs/recipes/WritingTests.md)
-  * [Usage with UI Frameworks](docs/recipes/UsageWithUIFrameworks.md)
-  * [Adding New Epics Asynchronously](docs/recipes/AddingNewEpicsAsynchronously.md)
-  * [Hot Module Replacement](docs/recipes/HotModuleReplacement.md)
+
+* [Cancellation](docs/recipes/Cancellation.md)
+* [Error Handling](docs/recipes/ErrorHandling.md)
+* [Injecting Dependencies Into Epics](docs/recipes/InjectingDependenciesIntoEpics.md)
+* [Writing Tests](docs/recipes/WritingTests.md)
+* [Usage with UI Frameworks](docs/recipes/UsageWithUIFrameworks.md)
+* [Adding New Epics Asynchronously](docs/recipes/AddingNewEpicsAsynchronously.md)
+* [Hot Module Replacement](docs/recipes/HotModuleReplacement.md)
 
 ## Help
-  * [FAQ](docs/FAQ.md)
-  * [Troubleshooting](docs/Troubleshooting.md)
+
+* [FAQ](docs/FAQ.md)
+* [Troubleshooting](docs/Troubleshooting.md)
 
 ## [API Reference](docs/api/SUMMARY.md)
-  * [createEpicMiddleware](docs/api/createEpicMiddleware.md)
-  * [combineEpics](docs/api/combineEpics.md)
-  * [EpicMiddleware](docs/api/EpicMiddleware.md)
+
+* [createEpicMiddleware](docs/api/createEpicMiddleware.md)
+* [combineEpics](docs/api/combineEpics.md)
+* [EpicMiddleware](docs/api/EpicMiddleware.md)
```

Comment:
I need to double check that this doesn't actually change the git-book generated output as I have a vague recollection that the indention mattered.